runtime/src/lib.rs

```diff
@@ -523,32 +578,50 @@ construct_runtime!(
         Aura: pallet_aura = 23,
         AuraExt: cumulus_pallet_aura_ext = 24,
 
+        // Scheduler
+        Scheduler: pallet_scheduler = 28,
+
+        // Preimage
+        Preimage: pallet_preimage = 29,
+
         // XCM helpers.
         XcmpQueue: cumulus_pallet_xcmp_queue = 30,
         PolkadotXcm: pallet_xcm = 31,
         CumulusXcm: cumulus_pallet_xcm = 32,
         MessageQueue: pallet_message_queue = 33,
 
+        // Contracts
         Contracts: pallet_contracts = 40,
+
+        // Nfts
+        Nfts: pallet_nfts = 50,
+        NftFractionalization: pallet_nft_fractionalization = 51,
+        Assets: pallet_assets::<Instance1> = 52,
```

Comment:
Perhaps `TrustBackedAssets` instead of `Assets`? Just thinking that when an additional instance gets added for `ForeignAssets` then the two are differentiated more?

Can probably also use the `TrustBackedAssets` type alias rather than `Instance1` so it reads more clearly.

I guess AH wasn't able to easily rename the existing `Assets` name whereas we have the opportunity to do it right.

Reply:
Unfortunately we don't get to choose the type alias name: https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/support/src/instances.rs

Reply:
I mean the pallet name within the runtime:

TrustBackedAssets: pallet_assets::<Instance1> = 52

So when we have another instance the names will be `TrustBackedAssets` and `ForeignAssets`. See the comment at https://github.com/polkadot-fellows/runtimes/blob/59172d5fb8feea2646490618bead3b2b4fb06220/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L280, whereby we dont have to worry about a breaking change in the name as its a new chain.